ci-operator/config/openshift/addon-operator/openshift-addon-operator-main.yaml

```diff
@@ -79,6 +79,8 @@ tests:
       OO_INSTALL_NAMESPACE: '!create'
       OO_PACKAGE: addon-operator
       OO_TARGET_NAMESPACES: '!all'
+    pre:
+    - chain: ipi-aws-pre
```

Comment:
We also need a post chain for cleanup, or we might leave resources.
Or is `ipi-aws-post` included automatically?

Reply:
Everything in the workflow is the same except for the missing pre-step https://github.com/openshift/release/blob/master/ci-operator/step-registry/optional-operators/ci/aws/optional-operators-ci-aws-workflow.yaml#L6.

Reply:
https://docs.ci.openshift.org/docs/architecture/step-registry/#ci-operator-test-configuration